webapp/channels/src/components/post/post_component.tsx

```diff
@@ -155,30 +155,6 @@ const PostComponent = (props: Props): JSX.Element => {
         }
     }, [a11yActive]);
 
-    useEffect(() => {
-        const handleAlt = (e: KeyboardEvent) => {
-            setAlt(e.altKey);
-        };
-        let removeEventListener: (type: string, listener: EventListener) => void;
-
-        document.addEventListener('keydown', handleAlt);
```

Comment:
We actually still need these key handlers so that the cursor changes to a pointer if the user presses alt with their mouse over the post, but we should only register them when the user is actually hovering over the post

Reply:
Hopefully that makes sense. Instead of registering the keyup/keydown handlers for the post when it gets mounted like we had previously, we want to only register them while the user is hovering over the post. The old version of the post component worked like that: https://github.com/mattermost/mattermost-webapp/blob/release-7.0/components/post_view/post/post.jsx